### ISSUE TITLE
Minimum required node version

### ISSUE THREAD

**Issue by Cael6** [UnitedIncome/serverless-python-requirements#56]
I have recently been getting this error:

```
  Serverless Error ---------------------------------------
 
  Serverless plugin "serverless-python-requirements" not found. Make sure it's installed and listed in the "plugins" section of your serverless config file.
 
  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
     Forums:        forum.serverless.com
     Chat:          gitter.im/serverless/serverless
 
  Your Environment Information -----------------------------
     OS:                     linux
     Node Version:           5.3.0
     Serverless Version:     1.19.0
```

I was able to identify that it is caused by a destructuring assignment which I believe is only natively available in node v6+. 

The README.md should be updated to show the minimum required version of node.

**Comment by dschep** [comment]
UGGH.. why the hell doesn't @npm do more than just warn about the `engines` option in `package.json`. Readme is probably the best way to do this, or just remove unsupported features.

**Comment by dschep** [comment]
Any one finding this issue: :+1: this comment if you want to be able to use node v4. (The v6 features aren't essential and could be worked around)

**Comment by dschep** [comment]
Closing this because serverless itself includes dependencies that depend on node 6:
```
npm WARN engine @serverless/fdk@0.3.5: wanted: {"node":">=6.10.0"} (current: {"node":"4.8.4","npm":"2.15.11"})
npm WARN engine punycode@2.1.0: wanted: {"node":">=6"} (current: {"node":"4.8.4","npm":"2.15.11"})
```

**Comment by dschep** [comment]
I've released this in version `3.0.0-beta`. It's a major version because the no deploy change is a breaking change and it's a beta because @UnitedIncome is nearing a public launch so I don't wanna pull a bunch of changes into our release.
